docs/application/dotnet/tutorials/certificates/managing-certificate-profile.md

```diff
@@ -34,7 +34,7 @@ Follow the same steps to create a certificate profile as described in `4. Create
 
 2. Use the existing author certificate
 
-    Click **No** to use the existing author certificate then it moves to `1.Create new or select existing distributor certiicate` of [Create a New Distributor Certificate](creating-certificates.md#create-a-new-distributor-certificate) to register DUID.
+    Click **No** to use the existing author certificate then it moves to [1.Create new or select existing distributor certiicate](creating-certificates.md#create_new_or_select_existing_distributor_certificate) to register DUID.
```

Comment:
Suggestion:
If you click **No** to use the existing author certificate then move to [create new or select existing distributor certificate](creating-certificates.md#create_new_or_select_existing_distributor_certificate) to register DUID.
Please verify.

Reply:
done